instrumentation-security/servlet-5.0/src/main/java/com/nr/instrumentation/security/servlet5/HttpServletHelper.java

```diff
@@ -105,4 +112,33 @@ public static void releaseServletLock() {
     private static String getNrSecCustomAttribName() {
         return NR_SEC_CUSTOM_ATTRIB_NAME + Thread.currentThread().getId();
     }
+
+    public static void gatherURLMappings(ServletContext servletContext) {
+        try {
+            Map<String, ? extends ServletRegistration> servletRegistrations = servletContext.getServletRegistrations();
+            getJSPMappings(servletContext, SEPARATOR);
+
+            for (ServletRegistration servletRegistration : servletRegistrations.values()) {
+                for (String s : servletRegistration.getMappings()) {
+                    URLMappingsHelper.addApplicationURLMapping(new ApplicationURLMapping(WILDCARD, s));
+                }
+            }
+        } catch (Exception ignored){
+        }
+    }
+
+    public static void getJSPMappings(ServletContext servletContext, String dir) {
+        try {
+            if(dir.endsWith(SEPARATOR)){
```

Comment:
Will this JSP mappings work on windows platform as well? as the SEPARATOR is set to "/"

Reply:
Yes, this is working on windows platform as well.